test/ninja/ninja_test_sconscripts/sconstruct_force_scons_callback

```diff
@@ -0,0 +1,11 @@
+SetOption('experimental','ninja')
+DefaultEnvironment(tools=[])
+
+env = Environment()
```

Comment:
add tools=[] here as well?

Reply:
fixed in a49c103

Reply:
well actually this compiles a c program. I can reconstruct the test so it doesnt need tools. This feature is not tied to any toolset

Reply:
If it's fairly easy to change to not needing a compiler, that'd be good.